cirro/cli/controller.py

```diff
@@ -46,9 +47,13 @@ def run_list_datasets(input_params: ListArguments, interactive=False):
 
 
 def run_ingest(input_params: UploadArguments, interactive=False):
+    logger = _get_logger()
     _check_configure()
+    logger.info("Instantiating Cirro client")
     cirro = DataPortalClient()
+    logger.info("Listing available projects")
     projects = cirro.project.list()
+    logger.info("Listing available processes")
```

Comment:
I don't know how I feel about these first three loggers

Reply:
The value I see is that they give the user some context as to why their terminal will be non-responsive for a second or two. I'm open to other approaches for messaging.

Reply:
project list and process list should both be under 500 ms total. When running from the cluster I do see a longer lag but I belive it happens before it hits that part of the code.

Reply:
What if we just logged something when the CLI launches like `"Collecting data from cirro.bio"`?

Reply:
Sorry I didn't follow up with this... Yeah that log message is fine. Make sure to use the base_url config property if you can though.